jaggeryapps/admin/conf/toplink_menu.json

```diff
@@ -0,0 +1,84 @@
+{
+  "Identity":{
+    "url": "https://cloudmgt.cloud.wso2.com/cloudmgt/site/pages/index.jag"
```

Comment:
Do we need to add the wso2 production urls to this?
Ideally these should be applied from the puppet and these values should be kept in hieradata and changed according to the environment.